src/components/UsernameEntry/UsernameEntry.js

```diff
@@ -23,12 +23,17 @@ class UsernameEntry extends React.Component {
 					Please select your slack handle to send a reminder
 				</div>
 				<div>
-					<ButtonList
-						items={this.props.users}
-						handleClick={(id, name) => this.sendReminder(name)}
-					/>
+					{this.props.users.length !== 0 && (
+						<ListSelection
+							items={this.props.users.map(user => ({
+								name: user.name,
+								id: user.name,
+								image: user.profile['image_48']
+							}))}
+							onClick={name => this.sendReminder(name)}
```

Comment:
Could simplify this to just pass in the reference, `onClick={this.sendReminder}`